packages/cli/src/util/dev/parse-query-string.ts

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Since this implementation does not preserve the order of the params, and assuming we care, perhaps there should be a TODO stating the limitation.